README.md

```diff
@@ -1,2 +1,81 @@
 # mqtt2http
-MQTT broker with hooks to send authenticate and publish events as HTTPs requests.
+
+MQTT broker with hooks to send authentication and publish events as HTTP requests.
+
+This MQTT broker receives MQTT messages with a topic on port 1883 and forwards them as POST requests to an HTTP service (see `MQTT2HTTP_PUBLISH_URL`). 
+The topic is either included in the URL or forwarded as the HTTP header X-Topic (see `MQTT2HTTP_TOPIC_HEADER` for customization).
+For any MQTT message that includes authentication information, the broker sends an HTTP Basic request to `MQTT2HTTP_AUTHORIZE_URL` to validate the username:password.
+
+There is also a small web interface available at `http://localhost:8080/publish?topic=...` (see `MQTT2HTTP_HTTP_LISTEN_ADDRESS` to customize the port). 
+The service requires HTTP Basic authentication. If authentication succeeds, the body is forwarded just like an MQTT message to `MQTT2HTTP_PUBLISH_URL` with the given topic.
```

Comment:
Because of #5 - if the publish URL is meant to be a http2mqtt service, then, this sentence is wrong